frame.py

```diff
@@ -0,0 +1,34 @@
+import os
+import urllib2
+from const import SUPPORTED_FILE_TYPES
+
+HASH_NOTEPAD = str(os.environ.get("HASH_NOTEPAD"))
+HASH_PAINT = str(os.environ.get("HASH_PAINT"))
+
+
+def file_exist(file_url):
+    """
+    Method that check if file at provided url exist.
+    :param file_url: Url of the file that should be checked.
+    :return: True if file exist, otherwise False.
+    """
+    request = urllib2.Request(file_url)
+    request.get_method = lambda: 'HEAD'
+    try:
+        response = urllib2.urlopen(request)
+        return True
+    except urllib2.HTTPError:
+        return False
+
+
+def get_hash(file_url):
```

Comment:
Have you considered adding support for more file types?
I.e. your env variable can have map of extension -> hash so you can add support for more files without changing code? In that case your function would be more abstract. BTW, you'll need to validate that map in service booting phase.